react-native/react/native/remote-component-loader.js

```diff
@@ -49,6 +50,14 @@ class RemoteComponentLoader extends Component {
 
     const componentToLoad = getQueryVariable('component')
     const selectorParams = getQueryVariable('selectorParams')
+    let title = getQueryVariable('title')
```

Comment:
I noticed in linux that it will say what the window title is, so might as well make it informative.

and since we have the title let's use that for the description of what the remote component is